backend/.env.example

```diff
@@ -14,3 +14,8 @@ SALT_ROUND=10  # number of rounds for salt generation. Default 10.
 # JWT
 JWT_SECRET=XXX  # random sercet to sign the JWT, NEED TO BE CHANGED FOR SECURITY REASONS
 JWT_EXPIRATION="3d"  # time before expiration of JWT, default 3 days
+
+# FILE UPLOAD
+MAX_TITLE=500  # max length of the article title, default 500b
+MAX_SUMMARY=500  # max length of the article summary, default 500b
+MAX_BODY=30000  # max length of the arcitle body, defautl 3kb
```

Comment:
Good change! Was going to suggest a max size for typed in another issue, but it is also nice to address it right now